apps/scratch/src/pages/__tests__/Solve.spec.tsx

```diff
@@ -261,4 +379,20 @@ test.describe("/solve/sessionId/taskId", () => {
 
     expect(page.taskConfigForm).toHaveCount(0);
   });
+
+  test("removing initial blocks does not increase the limit", async ({
+    page: pwPage,
+  }) => {
+    const page = new TestTaskPage(pwPage);
+
+    const { moveSteps, turnRight } = page.enabledBlockConfigButtons;
+
+    await expect(page.taskBlocks.catActor.editableBlock).toHaveCount(1);
```

Comment:
fewer magic numbers, more constants would increase readability and maintainability when we need to change the testing task definition. As per the comment somewhere else, if we generate the test task, that would kill 2 birds with 1 stone.

This being said, very clean tests ❤️ 

PS: don't harm birds, they're innocent 😁